Tests/NIOTests/ControlMessageTests.swift

```diff
@@ -90,4 +90,29 @@ class ControlMessageTests: XCTestCase {
         }
         XCTAssertEqual(expected, decoded)
     }
+
+    private func assertBuffersNonOverlapping(_ b1: UnsafeMutableRawBufferPointer,
+                                             _ b2: UnsafeMutableRawBufferPointer,
+                                             file: StaticString = #file,
+                                             line: UInt = #line) {
```

Comment:
No point passing these around if we aren't going to use them.

Reply:
Ahh yes, will start using them.